File: Beelina.APP/src/app/_models/results/product-withdrawal-entry-result.ts
import { IModelNode } from 'src/app/_interfaces/imodel-node';
import { ProductWithdrawalEntry } from '../product-withdrawal-entry';
import { IProductWithdrawalEntryPayload } from 'src/app/_interfaces/payloads/iproduct-withdrawal-entry-query.payload';
import { ProductStockAuditsResult } from './product-stock-audit-result';

export class ProductWithdrawalEntryResult extends ProductWithdrawalEntry implements IProductWithdrawalEntryPayload, IModelNode {
  public typename: string;
  public productWithdrawalAuditsResult: Array<ProductStockAuditsResult>;

  constructor() {
    super();
    this.productWithdrawalAuditsResult = [];
  }
}

File: Beelina.LIB/GraphQL/Results/ProductWithdrawalEntryResult.cs
using Beelina.LIB.Interfaces;
using Beelina.LIB.Models;

namespace Beelina.LIB.GraphQL.Results
{
    public class ProductWithdrawalEntryResult
        : ProductWithdrawalEntry, IProductWithdrawalEntryPayload
    {
        public List<ProductWithdrawalAuditResult> ProductWithdrawalAuditsResult { get; set; }

        public ProductWithdrawalEntryResult()
        {
            ProductWithdrawalAuditsResult = [];
        }
    }
}